packages/playground/src/dashboard/dao_view.vue

```diff
@@ -355,14 +357,21 @@ function filteredProposals(proposals: DaoProposalDetails[] | undefined) {
 }
 async function castVote() {
   loadingVote.value = true;
-
   if (grid) {
     try {
       await grid.dao.vote({
         farmId: selectedFarm.value,
         approve: castedVote.value,
         hash: selectedProposal.value,
       });
+      const { ayes, nays, ayesProgress, nayesProgress } = await grid.dao.getProposalVotes(selectedProposal.value);
+      const updatedProposal = proposals.value?.active.find(proposal => proposal.hash === selectedProposal.value);
+      if (updatedProposal) {
+        updatedProposal.ayes = ayes;
+        updatedProposal.nayes = nays;
```

Comment:
please just make all "nays" to "nayes" for consistency

Reply:
will leave it for now cuz it will take time as it's coming from the chain

Reply:
Actually no, "nayes" is what is returned from the chain so this change won't take time.

Reply:
@AhmedHanafy725 can you verify, please